.github/workflows/azure-ansible.yml

```diff
@@ -19,11 +19,11 @@ jobs:
       - name: (HELPER) Checkout Code
         uses: actions/checkout@v2
       
-      - name: Lint Role 
-        uses: github/super-linter@v3
-        env:
-          VALIDATE_ALL_CODEBASE: false
-          VALIDATE_ANSIBLE: true
-          ANSIBLE_DIRECTORY: /cloud_Azure/ansible/roles/kentik_az
-
-    
+      - name: Lint Role
+# TODO(lwolanin) reenable this before making any changes to azure ansible scripts
```

Comment:
maybe shortly explain why this is commented out

Reply:
I generally prefer unused code/config to be removed as opposed to commented out, unless it contains information useful for understanding why the used code/config exists or where it is heading. In all other cases it just pollutes searches. This is case is as benign as it gets, so I'm fine with it.

Reply:
@lukaa12 Please create a JIRA ticket to re-enable the linter

Reply:
DONE (https://clkentik.atlassian.net/browse/KNTK-220?atlOrigin=eyJpIjoiZDA3YmJiZmI4Y2E1NDE1Zjk4ZGE3NWY0YTk4MzNlNTkiLCJwIjoiaiJ9)